package.json

```diff
@@ -66,19 +67,18 @@
     "proxyquire": "^2.0.1",
     "pumpify": "^2.0.0",
     "puppeteer": "^5.0.0",
+    "request": "^2.88.2",
```

Comment:
Heh, why did this sneak in here?

Reply:
Because `retry-request` does `require("request")` (in some cases) and it makes Webpack print a warning. It's probably never actually used, but I just wanted to have a clean webpack run.

Reply:
Maybe it's better to explicitly ignore "request" package? Would not be a good idea to bundle such big package.
https://bundlephobia.com/result?p=request@2.88.2

Reply:
It's in `devDependencies` and won't be bundled (I hope so). But I'll make sure not to include it, yes.